packages/@okta/vuepress-site/docs/guides/registration-inline-hook/prepare-your-external-service/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Prepare your External Service
+---
+The external service is customer-provided software that Okta calls when the Registration Inline Hook fires. In this example, the external service is hosted on a browser-based web application service called [Glitch.me](https://glitch.me), which can be configured quickly to host the code that interacts with the Okta API calls.  //BD - Maybe include a link to the code for user to remix, i.e., copy to their account for testing.
+
+You can remix or copy the code at the following Glitch.me project: tbd
+
+The following Node.js content is required:
+
+- Node.js dependencies. For further information, see [Dependencies](/docs/guide/password-import-hook/nodejs/dependencies)
+- An HTTPS server. For further information, see [Implement an HTTPS server](/docs/guide/password-import-hook/nodejs/web-server)
```

Comment:
If you're using Glitch, isn't this taken care of already?

Reply:
Yes. This content can be consolidated or explained in a Glitch topic.